src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs

```diff
@@ -85,15 +81,19 @@ private sealed class State
             // Set once stream have been shutdown.
             public readonly TaskCompletionSource ShutdownCompletionSource = new TaskCompletionSource(TaskCreationOptions.RunContinuationsAsynchronously);
 
+            public State()
+            {
+                SendBuffers = new MsQuicBuffers();
```

Comment:
Putting `= new MsQuicBuffers()` in the field initializer triggers "Don't assign default values" or similar warning, and leaving it uninitialized breaks because the `MsQuicBuffers._handles` is null.